libr/anal/sign.c

```diff
@@ -67,6 +67,12 @@ R_API RList *r_sign_fcn_vars(RAnal *a, RAnalFunction *fcn) {
 	return ret;
 }
 
+R_API RList *r_sign_fcn_types(RAnal *a, RAnalFunction *fcn) {
```

Comment:
why have this wrapper function if its not providing anything ? why not just call r_anal_types_from_fcn ?

Reply:
I didn't know if we can fetch more types from a function aside from the ones from txf. If the current implementation is enough it's obvious that we can just call r_anal_types_from_fcn from addFcnTypes

Reply:
well, the function name seems exactly the same, but with twisted order of words. so i would avoid that wrapper for now